tests/src/Interop/WinRT/NativeComponent/Component.Contracts.BindingViewModel.h

```diff
@@ -0,0 +1,161 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#pragma once
+
+#include "Component.Contracts.BindingViewModel.g.h"
+#include <vector>
+
+template<typename T>
+struct BindableIteratorWrapper : winrt::implements<BindableIteratorWrapper<T>, winrt::Windows::UI::Xaml::Interop::IBindableIterator>
+{
+    BindableIteratorWrapper(winrt::Windows::Foundation::Collections::IIterator<T>&& iterator)
+        :m_iterator(std::move(iterator))
```

Comment:
`std::move()` isn't needed with a `&&`.

Reply:
The `std::move` is required here. Once you have a named r-value reference, that name is an lvalue and needs to be `std::move`d to become an xvalue and be moved-from.